src/Microsoft.ApplicationInsights.AspNet/project.json

```diff
@@ -15,18 +15,15 @@
         "Microsoft.AspNet.Http": "1.0.0-beta4",
         "Microsoft.AspNet.Http.Extensions": "1.0.0-beta4",
         "Microsoft.AspNet.Mvc.Core": "6.0.0-beta4",
-        "System.Net.NameResolution": "4.0.0-beta-*"
+        "System.Net.NameResolution": "4.0.0-beta-*",
```

Comment:
Maybe this dependency may go to dnxcore only. Dns should be defined in full framework by default